pkgs/development/compilers/openjdk/8.nix

```diff
@@ -198,12 +218,7 @@ let
       ln -s $jre/lib/openjdk/jre $out/jre
     '';
 
-    # FIXME: this is unnecessary once the multiple-outputs branch is merged.
     preFixup = ''
-      prefix=$jre stripDirs "$STRIP" "$stripDebugList" "''${stripDebugFlags:--S}"
-      patchELF $jre
-      propagatedBuildInputs+=" $jre"
```

Comment:
The JRE inputs need to be propagated to jdk as well to ensure that the `setJavaClassPath` hook is used.
This breaks for instance scalafmt[1] and scalafix[2].

* https://hydra.nixos.org/build/100896633
* https://hydra.nixos.org/build/100895668

related to ZHF #68361